src/dts/logger.d.ts

```diff
@@ -81,12 +82,3 @@ export declare class Logger implements ZLUX.Logger {
     private replayPatternsOnLogger;
     makeComponentLogger(componentName: string, messages?: MessageTable): ComponentLogger;
 }
- /*	
```

Comment:
Was this intentional? (I remember this being done automatically after stuff building or something)

Reply:
(I meant line 84 and down)

Reply:
It auto-generates and so its annoying to replace the header AND footer, so I kept just the header. There is no rule that says we need both, just one.